community-containers/fail2ban/fail2ban.json

```diff
@@ -30,6 +30,11 @@
                     "source": "nextcloud_aio_jellyfin",
                     "destination": "/jellyfin",
                     "writeable": false
+                },
+                {
+                    "source": "nextcloud_aio_jellyseerr",
```

Comment:
Thanks, please update also the readme for failwban that it also covers jellyseerr if installed :)